### PR TITLE
[AIRFLOW-2429] Fix contrib folder's flake8 errors

### PR DESCRIPTION
**Pull request by kaxil**
Make sure you have checked _all_ steps below.

### JIRA
- [x] My PR addresses the following [Airflow JIRA](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
    - https://issues.apache.org/jira/browse/AIRFLOW-2429
    - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a JIRA issue.

### Description
- [ ] Here are some details about my PR, including screenshots of any UI changes:

### Tests
- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits
- [x] My commits all reference JIRA issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

### Documentation
- [x] In case of new functionality, my PR adds documentation that describes how to use it.
    - When adding new operators/hooks/sensors, the autoclass documentation generation needs to be added.

### Code Quality
- [x] Passes `git diff upstream/master -u -- "*.py" | flake8 --diff`

### CODE REVIEW

**Comment by kaxil** [comment]
@Fokko PTAL

File: airflow/contrib/auth/backends/ldap_auth.py [hunk 1]
```diff
@@ -191,11 +192,11 @@ def try_login(username, password):
             username
         )
 
-        search_scopes = {
-            "LEVEL": LEVEL,
-            "SUBTREE": SUBTREE,
-            "BASE": BASE
-        }
+        # search_scopes = {
```
Comment (Fokko) [inline airflow/contrib/auth/backends/ldap_auth.py]: Maybe just remove this completely.

File: airflow/contrib/hooks/ftp_hook.py [hunk 2]
```diff
@@ -250,7 +250,7 @@ def get_conn(self):
             params = self.get_connection(self.ftp_conn_id)
 
             if params.port:
-               ftplib.FTP_TLS.port=params.port
+                ftplib.FTP_TLS.port = params.port
```
Comment (Fokko) [inline airflow/contrib/hooks/ftp_hook.py:253]: Nice catch

**Review by Fokko — CHANGES_REQUESTED** [review]
One small nit, apart from that LGTM. Awesome work @kaxil

File: airflow/contrib/auth/backends/ldap_auth.py (hunk 1 above)
Comment (kaxil) [reply airflow/contrib/auth/backends/ldap_auth.py]: sure

**Comment by kaxil** [comment]
Done @Fokko . Merging this.

**Comment by feng-tao** [comment]
hey @kaxil , I noticed that the following flake8 errors still exist in the contrib folders:
```
(venv3) tfeng@tfeng-mbp143 ~/playground/incubator-airflow (flake8_p3) $ flake8 airflow/contrib/
airflow/contrib/operators/mlengine_prediction_summary.py:116:9: W503 line break before binary operator
airflow/contrib/operators/mlengine_prediction_summary.py:117:9: W503 line break before binary operator
airflow/contrib/operators/mlengine_prediction_summary.py:118:9: W503 line break before binary operator
airflow/contrib/operators/mlengine_prediction_summary.py:120:9: W503 line break before binary operator
airflow/contrib/operators/mlengine_prediction_summary.py:122:31: E226 missing whitespace around arithmetic operator
airflow/contrib/operators/mlengine_prediction_summary.py:157:9: E125 continuation line with same indent as next logical line
airflow/contrib/operators/mlengine_prediction_summary.py:160:9: F841 local variable '_' is assigned to but never used
airflow/contrib/operators/mlengine_prediction_summary.py:161:14: W503 line break before binary operator
airflow/contrib/operators/mlengine_prediction_summary.py:165:14: W503 line break before binary operator
airflow/contrib/operators/mlengine_prediction_summary.py:166:14: W503 line break before binary operator
airflow/contrib/operators/cassandra_to_gcs.py:257:69: F812 list comprehension redefines 'name' from line 249
airflow/contrib/operators/cassandra_to_gcs.py:268:54: F812 list comprehension redefines 'name' from line 260
airflow/contrib/operators/cassandra_to_gcs.py:268:60: F812 list comprehension redefines 'value' from line 260
airflow/contrib/auth/backends/kerberos_auth.py:35:1: F401 'airflow.settings' imported but unused
airflow/contrib/auth/backends/google_auth.py:23:1: F401 'flask_login.login_required' imported but unused
airflow/contrib/auth/backends/google_auth.py:23:1: F401 'flask_login.logout_user' imported but unused
airflow/contrib/auth/backends/google_auth.py:23:1: F401 'flask_login.current_user' imported but unused
airflow/contrib/auth/backends/google_auth.py:33:1: F401 'airflow.settings' imported but unused
airflow/contrib/auth/backends/ldap_auth.py:22:1: F401 'flask_login.login_required' imported but unused
airflow/contrib/auth/backends/ldap_auth.py:22:1: F401 'flask_login.logout_user' imported but unused
airflow/contrib/auth/backends/ldap_auth.py:28:1: F401 'ldap3.BASE' imported but unused
airflow/contrib/auth/backends/ldap_auth.py:65:5: E722 do not use bare except'
airflow/contrib/auth/backends/ldap_auth.py:97:5: E722 do not use bare except'
airflow/contrib/auth/backends/github_enterprise_auth.py:23:1: F401 'flask_login.login_required' imported but unused
airflow/contrib/auth/backends/github_enterprise_auth.py:23:1: F401 'flask_login.logout_user' imported but unused
airflow/contrib/auth/backends/github_enterprise_auth.py:23:1: F401 'flask_login.current_user' imported but unused
airflow/contrib/auth/backends/github_enterprise_auth.py:33:1: F401 'airflow.settings' imported but unused
airflow/contrib/hooks/aws_hook.py:75:9: E722 do not use bare except'
airflow/contrib/hooks/azure_data_lake_hook.py:67:16: F821 undefined name 'FileNotFoundError'
```

**Comment by kaxil** [comment]
@feng-tao . Some of them are false positives:

* The `mlengine_prediction_summary` ones are using beam api operators hence using pipe for chaining which causes this false positive. 
* The `auth/backends/` does require `flask_login.login_required` & `'flask_login.logout_user'`.

The PR passed the **Flake 8** tests: 
`https://travis-ci.org/apache/incubator-airflow/builds/381786346`

**Comment by kaxil** [comment]
Although if you can resolve the following lines, it would be good:

```
airflow/contrib/auth/backends/ldap_auth.py:65:5: E722 do not use bare except'
airflow/contrib/auth/backends/ldap_auth.py:97:5: E722 do not use bare except'
airflow/contrib/hooks/aws_hook.py:75:9: E722 do not use bare except'
```

**Comment by Fokko** [comment]
Thanks @kaxil for picking this up. Ignoring false positives is also relatively easy in Flake8: http://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html

Please add a comment that provides the reason why the error is being ignored. This will allow us to flake8 the whole project instead of the diffs 👍

**Pull request closed without merging**